hw21/js/script.js

```diff
@@ -0,0 +1,96 @@
+let ul = document.querySelector("#ul");
+let lastSelected = null;
+
+ul.addEventListener("mousedown", function (e) {
+  e.preventDefault();
+});
+
+ul.addEventListener("click", function (e) {
+  if (e.target == this) {
+    return false;
+  }
+
+  if (!e.ctrlKey) {
+    clearSelected(this.children);
+  }
+
+  if (e.ctrlKey) {
+    removeBacklight(e.target);
+  } else {
+    addSelected(e.target);
+  }
+
+  if (e.shiftKey) {
```

Comment:
Unfortunately, that flow works incorrectly. 
Some tips
- you need to detect the last selection in general (not only if shiftKey was selected )
- `clearSelected` function impact behavior with the shift key
- also selection with shift should work in both sides (not required)